pkg/mesalib-radeon

```diff
@@ -0,0 +1,76 @@
+[mirrors]
+https://mesa.freedesktop.org/archive/13.0.6/mesa-13.0.6.tar.xz
+
+[vars]
+filesize=9179236
+sha512=2455e044647f674c2f1dd524143b61b8630713fa929bbf6fe1c981436a9c08dc933553d638a22b32d2799afb77be746355b48b1b20af172141c5b63891e94441
+
+[deps]
+dri2proto
+dri3proto
+presentproto
+gettext
+glproto
+libdrm
+libxdamage
+libxext
+libxml2
+libxshmfence
+libxxf86vm
+eudev
+expat
+llvm-clang36
+libelf
+
+[deps.host]
+flex
+bison
+
+[build]
+if false ; then
+  # add mesaglut to deps when you need it
+  # currently this is non-functional as nothing in the glut src tree will
+  # be built, so it needs some additional makefile patches.
+  tar xf "$C"/MesaGLUT-7.9.2.tar.gz
+  mv Mesa-7.9.2/src/glut src/
+  mv Mesa-7.9.2/include/GL/* include/GL/
+fi
+
+archflags=
+[ "$A" = arm ] || [ "$A" = mips ] && archflags=--with-dri-drivers=swrast
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) --with-sysroot=$butch_root_dir"
+
+# workaround for missing endian-ness defines taken from alpine linux
+case "$A" in
+x86*) echo "#define LITTLEENDIAN_CPU" > tmp.h ;;
+arm*) echo "#define BIGENDIAN_CPU"    > tmp.h ;;
+esac
```

Comment:
on sabotage, this would catch only x86_64 in the above clause, since 386 arch is called i386 or i486, or ...
best to use what we already got:
```
pkg/hping3:[ $("$K"/endiancheck.sh) = "big" ] &&  endian=__BIG_ENDIAN_BITFIELD
```